client/src/App.jsx

```diff
@@ -1,18 +1,19 @@
 import { Outlet } from "react-router-dom";
 import { useState } from "react";
+import { ToastContainer } from "react-toastify";
 import NavBar from "./components/navbar/NavBar";
 
 import "./App.css";
+import "react-toastify/dist/ReactToastify.css";
 
 function App() {
   const [currentUser, setCurrentUser] = useState(null);
 
   return (
     <div className="app-container">
-      {
-        currentUser != null ? <NavBar user={currentUser}/> : <NavBar />
-      }
+      {currentUser != null ? <NavBar user={currentUser} /> : <NavBar />}
```

Comment:
you can setup this easily by defining a default value to your Navbar component.

ex: export default function ({user = null})
if you pass an user it's take the value you pass else it will take a default value (null here)